test/secure_client.js

```diff
@@ -153,5 +154,33 @@ describe('MqttSecureClient', function () {
         done()
       })
     })
+
+    it.only('should support SNI on the TLS connection', function (done) {
```

Comment:
This should not have landed as it has a `only` still in the tests.

Reply:
yes I'll revert it, I just wasn't getting Travis CI to even identify the branch to run testing.